.circleci/config.yml.jinja

```diff
@@ -163,9 +163,15 @@ jobs:
             - run:
                   name: Build OCaml
                   command: ./scripts/skip_if_only_frontend.sh bash -c 'eval `opam config env` && make build 2>&1 | tee /tmp/artifacts/buildocaml.log'
+                  {%- if profile in medium_curve_profiles %}
+                  no_output_timeout: 20m
+                  {%- endif %}
             - run:
                   name: Build deb Package with keys
                   command: ./scripts/skip_if_only_frontend.sh make deb
+                  {%- if profile in medium_curve_profiles %}
+                  no_output_timeout: 20m
```

Comment:
It would be surprising if this run entry also needs this longer timeout, but it doesn't hurt much.